src/API/OnboardingTasks.php

```diff
@@ -833,19 +808,13 @@ public function undo_dismiss_task( $request ) {
 	 * @return WP_REST_Response|WP_Error
 	 */
 	public function snooze_task( $request ) {
-		$task_id         = $request->get_param( 'id' );
-		$task_list_id    = $request->get_param( 'task_list_id' );
-		$snooze_duration = $request->get_param( 'duration' );
-
-		$is_snoozeable = false;
+		$task_id      = $request->get_param( 'id' );
+		$task_list_id = $request->get_param( 'task_list_id' );
+		$duration     = $request->get_param( 'duration' );
 
-		$snooze_task = TaskLists::get_task( $task_id, $task_list_id );
+		$task = TaskLists::get_task( $task_id, $task_list_id );
 
-		if ( $snooze_task && isset( $snooze_task['isSnoozeable'] ) && $snooze_task['isSnoozeable'] ) {
-			$is_snoozeable = true;
-		}
-
-		if ( ! $is_snoozeable ) {
+		if ( ! $task || ! $task->is_snoozeable ) {
```

Comment:
May be subjective, but I mildly prefer writing this condition this way:

```suggestion
		if ( ! ( $task && $task->is_snoozeable )) {
```

Just a little clearer to me, but perhaps just me 🤷🏻‍♂️

Reply:
I think the convention used in core and WCA is typically this pattern and usually favors less parentheses so I'd recommend keeping the existing style here.